app/src/main/java/com/example/lawn_care/viewYourProperties.java

```diff
@@ -66,14 +66,20 @@ public void onResponse(String response) {
                                 //find the linear layout from the screen
                                 LinearLayout linearLayout=findViewById(R.id.LL_yourProperties);
 
+
                                 //0 to len-1, the first index is the success check, the rest are listings, but the first address starts at 0, so listings go from 0 to n-1
                                 for(int x = 0; x < jsonResponse.length()-1; x++){
                                     //create a new linear layout so each listing can be in a view, easier to do stuff with
                                     LinearLayout listingItem = new LinearLayout(viewYourProperties.this);
+
                                     listingItem.setOrientation(LinearLayout.VERTICAL);
                                     //creates two views, one for address the other for work needed
                                     TextView TV_address=new TextView(viewYourProperties.this);
                                     TextView TV_workNeeded=new TextView(viewYourProperties.this);
+                                    //Place down the buttons for edit and delete
+                                    Button deleteButton=new Button (viewYourProperties.this);
+                                    Button editButton=new Button (viewYourProperties.this);
```

Comment:
prepend views with type, for example, prepend buttons with BTN_

Reply:
I changed what you requested.